routers/api/v1/repo/pull.go

```diff
@@ -456,6 +468,18 @@ func EditPullRequest(ctx *context.APIContext, form api.EditPullRequestOption) {
 			ctx.Error(http.StatusInternalServerError, "ReplaceLabelsError", err)
 			return
 		}
+
+		if ctx.Repo.Owner.IsOrganization() {
+			labels, err = models.GetLabelsInOrgByIDs(ctx.Repo.Owner.ID, form.Labels)
```

Comment:
To Check: does this overwrite repo lables?

Reply:
Good catch, I believe it will.

Reply:
I was not sure thats why I didnt approved/disaproved - had to look at the code 
It did